packages/babel-plugin-proposal-record-and-tuple/test/index.js

```diff
@@ -1,3 +1,7 @@
 import runner from "@babel/helper-plugin-test-runner";
```

Comment:
Can we just remove and deprecate this plugin? I can add it to the babel/babel-archive repo.

Reply:
Most archived libraries are either trivial syntax plugins or helpers deemed no longer used by us. Like I said although this plugin can be archived now, it can be repurposed to a codemod that potentially helps people migrate from the novel syntax to the Record/Tuple factory calls, we should guarantee that this plugin works for every `@babel/*` 7 versions, as long as the parser supported R&T.

This plugin depends on `@babel/helper-module-imports` and `@babel/helper-validator-option`. It is not a very trivial plugin like those archived syntax plugins. Staying in the monorepo gives us the assurance that it won't break when users are migrating from Babel 7 to Babel 8.